integration_tests/package.json

```diff
@@ -25,5 +26,8 @@
     "mocha": "^8.2.0",
     "ramda": "^0.27.1",
     "typescript": "^4.0.5"
+  },
+  "dependencies": {
+    "google-protobuf": "^3.15.8"
```

Comment:
wouldn't this also be a `devDependencies`?

Reply:
Yes. I fixed this.